cli/commands/benchmark-server-timing.mjs

```diff
@@ -206,10 +210,20 @@ function outputResults( opt, results ) {
 		percentiles.forEach( ( percentile ) => {
 			headings.push( `Response Time (p${ percentile })` );
 		} );
+
+		if ( opt.showVariance ) {
+			headings.push( 'Response Time (SD)' );
+			headings.push( 'Response Time (IQR)' );
+		}
+
 		Object.keys( allMetricNames ).forEach( ( metricName ) => {
 			percentiles.forEach( ( percentile ) => {
 				headings.push( `${ metricName } (p${ percentile })` );
 			} );
+			if ( opt.showVariance ) {
+				headings.push( `${ metricName } (SD)` );
+				headings.push( `${ metricName } (IQR)` );
+			}
 		} );
 	} else {
```

Comment:
Can you add this to the `else` right below here as well? Otherwise, the `showVariance` would only work when also enabling `showPercentiles`, which I don't think should be needed.

It may actually lead to an error right now because this is missing. Have you tested this without `showPercentiles` but with `showVariance`?

Reply:
Good catch, I hadn't tested this without percentiles, it seems. Should be all good now.